### PR TITLE
Document Android emulator usage

### PR DESCRIPTION
**Pull request by dbnicholson**
I've found that developing against the Android emulator results in a much more streamlined development experience. This documents how to use the emulator without Android Studio.

https://phabricator.endlessm.com/T33535

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,3 +114,142 @@ You could also do so using [Weinre](https://people.apache.org/~pmuellr/weinre/do
...
+## Using the Android Emulator
+
+Installing the APK to a real device during development is slow. Instead, the
+[Android Emulator](https://developer.android.com/studio/run/emulator) can be
+used for faster installs in a clean environment. The recommended way to use
+the emulator is through Android Studio, but it can also be invoked directly.
+
+First, ensure the emulator is installed in the Android SDK. The `make setup`
+target will install the necessary pieces. Assuming the SDK is installed in
+`~/.android/sdk`:
```
Comment (manuq) [inline README.md]: From the toolbox created with `make build_toolbox` this path is `/opt/android/sdk/`.

File: README.md [hunk 2]
```diff
@@ -114,3 +114,142 @@ You could also do so using [Weinre](https://people.apache.org/~pmuellr/weinre/do
...
+
+```
+~/.android/sdk/cmdline-tools/latest/bin/avdmanager list device
+```
+
+To create an AVD, a name, system image and device must be provided. Assuming
+the system image provided above and a Pixel 5 device:
+
+```
+~/.android/sdk/cmdline-tools/latest/bin/avdmanager create device --name test \
```
Comment (manuq) [inline README.md]: In my case from toolbox this was `create avd` not `create device`.
```suggestion
~/.android/sdk/cmdline-tools/latest/bin/avdmanager create avd --name test \
```

**Review by manuq — APPROVED** [review]
Looks great. This doesn't work from toolbox:

```
$ /opt/android/sdk/emulator/emulator -avd test
INFO    | Android emulator version 31.2.10.0 (build_id 8420304) (CL:N/A)
/opt/android/sdk/emulator/qemu/linux-x86_64/qemu-system-x86_64: error while loading shared libraries: libpulse.so.0: cannot open shared object file: No such file or directory
```

But I guess that's expected. Please check my suggestion, Either it's a typo or my SDK version in toolbox is not the same as the one installed by `make setup`.

File: README.md (hunk 2 above)
Comment (dbnicholson) [reply README.md]: Oops, that is correct.

File: README.md (hunk 1 above)
Comment (dbnicholson) [reply README.md]: Hmm, that might be better. You do need to be root to install SDK components then, but it would be more consistent.

**Comment by dbnicholson** [comment]
> /opt/android/sdk/emulator/qemu/linux-x86_64/qemu-system-x86_64

That's interesting. I would have figured they'd bundle all the dependencies, but I guess not. You could try installing `qemu-system-x86` in the container. That would hopefully get you all the same package dependencies.

**Comment by dbnicholson** [comment]
I was able to run `emulator` in toolbox once I installed the QEMU packages, so I added those to the toolbox image.

**Comment by manuq** [comment]
retest this please

**Pull request merged**